pkg/sshd/etc/profile

```diff
@@ -0,0 +1,13 @@
+export CHARSET=UTF-8
+export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+export PAGER=less
+export PS1='(sshd ns):\w\$ '
+export PS1='\h:\w\$ '
```

Comment:
I was proposing to merge the sources (i.e. `pkg/getty` and `pkg.sshd`) but to still produce two distinct output containers which could be added independently.